<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Segmentationwidget</class>
 <widget class="QWidget" name="Segmentationwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>245</width>
        <height>477</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBoxPathFunction">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>200</height>
          </size>
         </property>
         <property name="title">
          <string>Path Propagation Function</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0" colspan="3">
           <widget class="QSplitter" name="splitter_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>81</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <widget class="QRadioButton" name="pfmaxgeo">
             <property name="text">
              <string>&amp;Geodesic Restriction Function</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
            <widget class="QRadioButton" name="pfmax">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Ma&amp;xFunction</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
            <widget class="QRadioButton" name="pfsum">
             <property name="text">
              <string>SumFunction</string>
             </property>
            </widget>
           </widget>
          </item>
          <item row="1" column="0" colspan="3">
           <widget class="QSplitter" name="splitter_4">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>27</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QLabel" name="label">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>27</height>
              </size>
             </property>
             <property name="text">
              <string>Alpha:</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_2">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>27</height>
              </size>
             </property>
             <property name="text">
              <string>Beta:</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_3">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>27</height>
              </size>
             </property>
             <property name="text">
              <string>Thickness</string>
             </property>
            </widget>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QDoubleSpinBox" name="AlphaSpinBox">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>24</height>
             </size>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>-1.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="BetaSpinBox">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>24</height>
             </size>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="maximum">
             <double>4.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>0.500000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QDoubleSpinBox" name="thickDoubleSpinBox">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>24</height>
             </size>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="singleStep">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBoxGradient">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>80</height>
          </size>
         </property>
         <property name="title">
          <string>Gradients</string>
         </property>
         <widget class="QSplitter" name="splitter_5">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>303</width>
            <height>50</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QRadioButton" name="morphological_grad">
           <property name="text">
            <string>&amp;Morphological</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QRadioButton" name="sobel_grad">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Sobel</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>144</height>
          </size>
         </property>
         <property name="title">
          <string>Visualization and Execution</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QSplitter" name="splitter">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>27</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QPushButton" name="eraserButton">
             <property name="text">
              <string>Erase</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="autoExclusive">
              <bool>true</bool>
             </property>
            </widget>
            <widget class="QPushButton" name="drawButton">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>23</height>
              </size>
             </property>
             <property name="text">
              <string>Draw</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="autoExclusive">
              <bool>true</bool>
             </property>
            </widget>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="ClearButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>27</height>
             </size>
            </property>
            <property name="text">
             <string>Clear Seeds</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QSplitter" name="splitter_3">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>27</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QPushButton" name="pushButtonShowSeeds">
             <property name="text">
              <string>View Seeds</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
            <widget class="QPushButton" name="pushButtonShowMask">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>23</height>
              </size>
             </property>
             <property name="text">
              <string>View Mask</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="SegmentationButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string>Start Segmentation</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButtonSave">
     <property name="text">
      <string>Save Segmentation</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
